_data/communication.yml

```
@@ -180,6 +180,14 @@ websites:
       software: Yes
       doc: https://peerio.zendesk.com/hc/en-us/articles/202933829-How-do-I-setup-two-factor-authentication-
 
+    - name: Riot.im
```

On their website they use "Riot" as their name.
```suggestion
    - name: Riot
```